src/components/AppContent/AppContent.vue

```diff
@@ -24,14 +24,37 @@
 
 <template>
```

Comment:
Can you add a `<doc />` section here to describe how we can  and should use this component? :)
You can find examples on others components like https://github.com/nextcloud/nextcloud-vue/blob/5a3270028d4b958fe0d31866d9fdb7a15d8a036c/src/components/SettingsSection/SettingsSection.vue#L24-L41

Reply:
Done!